### PR TITLE
implemented disabled property for sulu field-type components

### PR DESCRIPTION
**Pull request by nnatter**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | yes
| BC breaks? | no
| Deprecations? | no
| Related issues/PRs | #3248 
| License | MIT

#### What's in this PR?

This PR implements a disabled property for all components that are used by field-type components.. Field-type components are the building blocks of forms and can be used inside of XML templates.

#### Why?

Our XML form templates already allow to set a disabledCondition on a property. To make use of this feature, we need to signify to the user that a field-type component is disabled.

### CODE REVIEW

File: src/Sulu/Bundle/AdminBundle/Resources/js/components/BlockCollection/SortableBlocks.js [hunk 1]
```diff
@@ -2,11 +2,13 @@
 import React from 'react';
```
Comment (nnatter) [inline src/Sulu/Bundle/AdminBundle/Resources/js/components/BlockCollection/SortableBlocks.js]: i noticed that this file is named `SortableBlocks.js` and the respective css file is named `sortableBlockList.scss`. should we rename this file to `SortableBlockList.js`?

File: src/Sulu/Bundle/ContentBundle/Resources/config/forms/PageExcerpt.xml [hunk 2]
```diff
@@ -18,12 +18,12 @@
                 <title>sulu_admin.description</title>
             </meta>
         </property>
-        <property name="categories" type="category_list">
+        <property name="categories" type="category_list" disabledCondition="title == 'test'">
```
Comment (danrot) [inline src/Sulu/Bundle/ContentBundle/Resources/config/forms/PageExcerpt.xml]: That should probably not be here, right? :slightly_smiling_face:

File: src/Sulu/Bundle/ContentBundle/Resources/config/forms/PageExcerpt.xml [hunk 3]
```diff
@@ -18,12 +18,12 @@
...
             <meta>
                 <title>sulu_category.categories</title>
             </meta>
         </property>
-        <property name="tags" type="tag_list">
+        <property name="tags" type="tag_list" disabledCondition="title == 'test'">
```
Comment (danrot) [inline src/Sulu/Bundle/ContentBundle/Resources/config/forms/PageExcerpt.xml]: Same as above

File: src/Sulu/Bundle/SecurityBundle/Resources/js/containers/RoleAssignments/RoleAssignment.js [hunk 4]
```diff
@@ -5,14 +5,19 @@ import {MultiSelect} from 'sulu-admin-bundle/components';
 import type {Localization} from 'sulu-admin-bundle/stores';
 import roleAssignmentStyle from './roleAssignment.scss';
```
Comment (danrot) [inline src/Sulu/Bundle/SecurityBundle/Resources/js/containers/RoleAssignments/RoleAssignment.js]: Maybe also add an opacity here for consistency?

File: src/Sulu/Bundle/AdminBundle/Resources/js/components/BlockCollection/SortableBlocks.js (hunk 1 above)
Comment (danrot) [reply src/Sulu/Bundle/AdminBundle/Resources/js/components/BlockCollection/SortableBlocks.js]: I would say that makes sense, yeah :slightly_smiling_face:

**Pull request merged**